kytos/core/kytosd.py

```diff
@@ -102,13 +105,35 @@ def main():
 
     config = KytosConfig().options['daemon']
 
+    # Configure to log uncaught exceptions to errlog file
+    sys.excepthook = exhandler
+
     if config.foreground:
         async_main(config)
     else:
         with daemon.DaemonContext():
             async_main(config)
 
 
+# pylint: disable=invalid-name
+def exhandler(exctype, value, tb):
```

Comment:
Please try to find a balance between long comprehensive names and short names, avoiding abbreviations as much as possible. `exhandler` is clear in the context of this PR, but not to someone who will be reading through the function names later.

`exc_handler` would be better, or even a full `exception_handler` name, as this is a catch-all function that may be called from anywhere in the code.

Same goes for variable names: `exctype` -> `exc_type`